### ISSUE TITLE
Implement long-polling in the UI for /poll_filter

### ISSUE THREAD

**Issue by simontuffs** [Pogadog/anomalizer-grafana-plugin#6]
/poll_filter is a long-polling API to return the result of /filter when it is changed on the server side.  Needs to be implemented in the plugin and stand-alone UI's.

Return schema is identical to /filter.

**Comment by pinky00707** [comment]
Indeed. This will require some research regarding Grafana preferences, but would be super cool when implemented.
